COPYRIGHT

```diff
@@ -0,0 +1,4 @@
+This project is licensed under MIT.
```

Comment:
I was wondering last days whether APL was the right one or not  for us. Honestly I can't say :(

Reply:
I find MIT much more straightforward, don't like all the APL stuff. In the end they are both permissive licenses, and I don't like to endorse their patent-related stuff

Reply:
+1 MIT then

Reply:
I don't want to support the idea that software patent can be legit, especially here in Europe...